common/lib/container-definitions/package.json

```diff
@@ -52,7 +52,7 @@
     "@fluidframework/eslint-config-fluid": "^0.28.2000-0",
     "@microsoft/api-extractor": "^7.22.2",
     "@rushstack/eslint-config": "^2.5.1",
-    "@types/mocha": "^8.2.2",
+    "@types/mocha": "^9.1.1",
```

Comment:
Curious that this package has a dependency on @types/mocha, but not mocha itself.  Try removing?

Reply:
It does appear that mocha is not used in this directory at all. Removed.